src/FlaUI.WebDriver/InputState.cs

```diff
@@ -1,12 +1,112 @@
-﻿namespace FlaUI.WebDriver
+﻿using System.Diagnostics;
+
+namespace FlaUI.WebDriver
 {
     public class InputState
     {
+        private readonly Dictionary<string, InputSource> _inputStateMap = new();
+
         public List<Action> InputCancelList = new List<Action>();
 
         public void Reset()
         {
             InputCancelList.Clear();
```

Comment:
Shouldn't `Reset` also clear the input state map?

Reply:
Hmm, can't see that in the spec. Removing an input source needs to assert that no item in the input cancel list [has an ID of the removed source](https://github.com/FlaUI/FlaUI.WebDriver/pull/33/files#diff-ee747bbb97900916c007399c8eb30825731f8e57333e52aff56ed85be5ced991R107) but i can't find the part of the spec where resetting the input cancel list is specified.

Reply:
The spec says that https://www.w3.org/TR/webdriver2/#release-actions should "reset the input state" which is removing the entire input state (for the "current top-level browsing context" which we interpret as the "current window"). So that certainly should clear the input sources. See also https://www.w3.org/TR/webdriver2/#dfn-reset-the-input-state

Reply:
Ah yes, got it 👍